### DISCUSSION TITLE
uvr5.6 cannot open on Mac Sonoma 14.3.1

### DISCUSSION THREAD

**Discussion by PurpleUnicornBJ** [Anjok07/ultimatevocalremovergui#1169] [Q&A]
I downloaded and installed V5.6. But the app cannot be opened as a pop-up shows up, saying due to apple safety policy, I cannot use it. What should I do???

**Comment by ahmedabdurrahman** [comment]
same problem!

**Reply by bitjeep** [reply]
Check Settings -> Privacy & Security and scroll down to where it says: "Allow applications downloaded from:". There should be an option for you to allow it to run there.

**Reply by PurpleUnicornBJ** [reply]
it won't work even though i choose [app store and recognized developers]

**Reply by PurpleUnicornBJ** [reply]
I can install the app into my Macbook but cannot open it when finishing installing. And the pop-up says 「Apple Inc. couldn't check whether it contains malware」

**Reply by Blend30** [reply]
Go back to Ventura

**Reply by bitjeep** [reply]
Screenshot of it working fine on Sonoma. I believe there was an extra step in the Privacy & Security settings where I had to explicitly allow it (beyond just the radio button setting). Try scrolling down to the bottom of those settings after you attempt to run it.

![uvr5_6_on_sonoma](https://github.com/Anjok07/ultimatevocalremovergui/assets/13810558/692945da-9a7a-41c2-b02c-074bf5e84fb8)

**Comment by cobalt107** [comment]
Download UVR 5.5 instead.

Go here: [https://github.com/Anjok07/ultimatevocalremovergui/releases](https://github.com/Anjok07/ultimatevocalremovergui/releases)

- Scroll down to v5.5
- Open "Assets" dropdown list (Do not click download link in main body - that will give you v5.6)
- Download file from Sept 29th 2023 (Ultimate_Vocal_Remover_v5_5_MacOS_x86_64_9_29.dmg)
- Delete/uninstall v5.6 from your computer if you haven't already done so
- Once v5.5 is installed, do not update
